test/cases/cleanups/suppressTypeErrors/index.test.ts

```diff
@@ -0,0 +1,9 @@
+import { describe, expect, it } from "vitest";
+import { runMutationForTest } from "../../../../src/tests/testSetup.js";
+
+describe("cleanups - suppressTypeErrors", () => {
+	it("matches snapshot", async () => {
```

Comment:
[Style] Since there are a bunch of these, and we're going to have to copy & paste them every time we want a new one... how about trimming down to just one `test`?

```suggestion
test("cleanups - suppressTypeErrors", async () => {
```